doc/source/train/user_guide.rst

```diff
@@ -393,53 +393,46 @@ Callbacks
 ~~~~~~~~~
 
 You may want to plug in your training code with your favorite experiment management framework.
-Ray Train provides an interface to fetch intermediate results and callbacks to process/log your intermediate results.
+Ray Train provides an interface to fetch intermediate results and callbacks to process/log your intermediate results
+(the values passed into ``train.report(...)``).
 
-You can plug all of these into Ray Train with the following interface:
+Ray Train contains built-in callbacks for popular tracking frameworks, or you can implement your own callback via the ``TrainCallback`` interface.
+
+.. _train-builtin-callbacks:
+
+Built-in Callbacks
+++++++++++++++++++
+
+The following ``TrainingCallback``\s are available and will log the intermediate results of the training run.
+
+1. :ref:`train-api-json-logger-callback`
+2. :ref:`train-api-tbx-logger-callback`
+3. :ref:`train-api-mlflow-logger-callback`
+
+Example usage on how to log intermediate results to MLflow and Tensorboard
 
 .. code-block:: python
 
     from ray import train
-    from ray.train import Trainer, TrainingCallback
+    from ray.train import Trainer
+    from ray.train.callbacks import MLflowLoggerCallback, TBXLoggerCallback
     from typing import List, Dict
 
-    class PrintingCallback(TrainingCallback):
-        def handle_result(self, results: List[Dict], **info):
-            print(results)
-
     def train_func():
         for i in range(3):
             train.report(epoch=i)
 
     trainer = Trainer(backend="torch", num_workers=2)
     trainer.start()
+
+    # Run the training function, logging all the intermediate results to MLflow and Tensorboard.
     result = trainer.run(
         train_func,
-        callbacks=[PrintingCallback()]
+        callbacks=[MLflowLoggerCallback(experiment_name="train_experiment"), TBXLoggerCallback()]
```

Comment:
Does this work out of the box? Would be helpful to handhold the user a little bit more here (e.g. include pip installations and opening up the browser to view results).

Reply:
Updated!